src/sidebar/activity/model/Activity.ts

```diff
@@ -1,6 +1,6 @@
 import User from "../../../common/domain/componentLike/model/User";
 import Version from "../../../common/domain/componentLike/model/Version";
-import { getDateAgo } from "../util/activityUtil";
+import { getDateAgo } from "../util/activityUi";
```

Comment:
Why `activityUtil` is renamed as `activityUi`? `activityUtil` seemed more proper 🤔

Reply:
I wanted to separate View and Model(Domain) logic throughout the project. I suffixed view logic helpers with "Ui", model logic helpers with "Util"